editor.planx.uk/src/@planx/components/List/model.ts

```diff
@@ -65,11 +66,28 @@ export type DateField = {
   data: DateInput & { fn: string };
 };
 
+export type MapField = {
+  type: "map";
+  data: {
+    title: string;
+    fn: "features";
```

Comment:
I _think_ in cases of the `"map"` field it will be helpful to have a static/predictable `fn` rather than user-configurabled free text `string` type if we want to do special  `handleSubmit` geojson formatting - any objections?

Reply:
Switched to generic `"string;"` for now then! 

Traditionally, I think it's come in very handy to have static "fn" for geo components like FindProp, Draw, Constraints etc - but we can always update this later based on feedback / better understanding of real scenarios (eg if there ever needs to be > 1 map field).